arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -10,12 +10,10 @@
     "https"
   ],
```

Comment:
It seems azure portal is using service bus RP's 'checkNameAvailability' API, this API is not documented in this swagger. Can we add this?